pandas/tests/test_categorical.py

```diff
@@ -1554,18 +1554,6 @@ def test_deprecated_labels(self):
             res = cat.labels
         self.assert_numpy_array_equal(res, exp)
 
-    def test_deprecated_levels(self):
```

Comment:
1) Whoops, I'll generate new pickles then.

2) Wait, but isn't that what the `test_pickle` tests are for?  I'm confused here...